doc/caveats.md

```diff
@@ -28,12 +28,11 @@ cross-platform notification of events in git repositories (e.g. pushes).
 Using such a mechanism, contributors could notify a project maintainer about
 changes in their public repositories, including issues or messages, via a public
 API. A maintainer could set up automated fetches of new issues, importing new
-issues and messages. This is, however unrelated and still work in progress.
+issues and messages. This is however, unrelated and still work in progress.
```

Comment:
My intuition tells me to put a comma before and after the "however".

Reply:
hmm... it starts to get complicated then..
What if we take the "however" to the front of the sentence?

Reply:
did that.